CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+
+# Contributing guidelines
+
+## Rendering notebooks
+
+This repository uses [snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html) to render the all notebooks.
+The `Snakefile` calls the `scripts/render-notebooks.R` which renders the notebooks but leaves these `.Rmd` files still be individually downloaded by a user and [ran without the `pandoc` error](https://github.com/AlexsLemonade/refinebio-examples/pull/148#issuecomment-669170681).=
+`snakemake` should be ran after changes have been made and before any `Pull Request` are filed.
+
+### How to re-render the notebooks
+
+**Step 1)** Install snakemake (if you haven't before).
+Follow the installation instructions on the [snakemake docs](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html).
+
+**Step 2)** Add any new `.Rmd` notebooks to `target:`, `input` section of the `Snakefile` as a file paths relative to the `Snakefile`.
```

Comment:
This line here is a bit confusing. Should the new `.Rmd` notebooks be added to the `target:` and `input` sections of the `Snakefile`? In other words, does this mean that the each of the notebooks should be added in two different places?

Reply:
I'm going to clarify and add to these instructions and let me know if it makes more sense!